bugs/gh-529-539.txt

```diff
@@ -0,0 +1,2734 @@
+      "resourceGroup": "srugroupIPV6"
+    }
+  ],
+  "loadBalancerInboundNatRules": null,
+  "name": "sruNic-xenial-v6",
+  "primary": null,
+  "privateIpAddress": "ace:cab:deca:deed::4",
+  "privateIpAddressVersion": "IPv6",
+  "privateIpAllocationMethod": "Dynamic",
+  "provisioningState": "Succeeded",
+  "publicIpAddress": null,
+  "resourceGroup": "srugroupIPV6",
+  "subnet": {
+    "addressPrefix": null,
+    "addressPrefixes": null,
+    "delegations": null,
+    "etag": null,
+    "id": "/subscriptions/12aad61c-6de4-4e53-a6c6-5aff52a83777/resourceGroups/srugroupIPV6/providers/Microsoft.Network/virtualNetworks/sruVnet-xenial/subnets/sruSubnet",
+    "ipConfigurationProfiles": null,
+    "ipConfigurations": null,
+    "name": null,
+    "natGateway": null,
+    "networkSecurityGroup": null,
+    "privateEndpointNetworkPolicies": null,
+    "privateEndpoints": null,
+    "privateLinkServiceNetworkPolicies": null,
+    "provisioningState": null,
+    "purpose": null,
+    "resourceGroup": "srugroupIPV6",
+    "resourceNavigationLinks": null,
+    "routeTable": null,
+    "serviceAssociationLinks": null,
+    "serviceEndpointPolicies": null,
+    "serviceEndpoints": null
+  },
+  "type": "Microsoft.Network/networkInterfaces/ipConfigurations",
+  "virtualNetworkTaps": null
+}
+{
+  "fqdns": "",
+  "id": "/subscriptions/12aad61c-6de4-4e53-a6c6-5aff52a83777/resourceGroups/srugroupIPV6/providers/Microsoft.Compute/virtualMachines/test-sru-xenial-advanced",
+  "location": "eastus2",
+  "macAddress": "00-0D-3A-DE-17-D7,00-0D-3A-DE-16-7E",
+  "powerState": "VM running",
+  "privateIpAddress": "10.0.0.4,ace:cab:deca:deed::4,10.0.0.5",
+  "publicIpAddress": "52.254.19.101",
+  "resourceGroup": "srugroupIPV6",
+  "zones": ""
+}
+Created test-sru-xenial-advanced: ubuntu@52.254.19.101
+
+status: done
+time: Wed, 09 Sep 2020 16:50:12 +0000
+detail:
+DataSourceAzure [seed=/dev/sr0]
+--- TEST: EXPECT NO KVP dump logs in cloud-init.log
+cloud-init	20.2-45-g5f7825e2-0ubuntu1~16.04.1
+sudo: unable to resolve host SRU-worked-azure
+{
+ "v1": {
+  "datasource": "DataSourceAzure [seed=/dev/sr0]",
+  "errors": []
+ }
+}
+Startup finished in 5.532s (kernel) + 24.126s (userspace) = 29.659s
+          7.326s cloud-init.service
+          4.374s cloud-init-local.service
+REBOOTING TO Confirm boot extra timeout across reboot LP: #1830740
+sudo: unable to resolve host SRU-worked-azure
+Connection to 52.254.19.101 closed by remote host.
+ssh: connect to host 52.254.19.101 port 22: Connection reset by peer
+Retrying in 5 seconds: ssh ubuntu@52.254.19.101 -- cloud-init status --wait --long 
+ssh: connect to host 52.254.19.101 port 22: Connection timed out
+Retrying in 5 seconds: ssh ubuntu@52.254.19.101 -- cloud-init status --wait --long 
+
+status: done
+time: Wed, 09 Sep 2020 16:51:25 +0000
+detail:
+DataSourceAzure [seed=/var/lib/waagent]
+--- TEST: BEGIN EXPECT Long boot time LP: #1830740
+Startup finished in 6.058s (kernel) + 11.818s (userspace) = 17.877s
```

Comment:
It's that value expected ? It seems way smaller than the other releases

Reply:
@lucasmoura good eyes. And yes I should have qualified the test message to mention xenial is unaffected (because we only render fallback network config on Xenial (dhcp on eth0). So it shouldn't affect this release.